submissions/onyinyeChukwuka/javascript/signup.js

```diff
@@ -0,0 +1,113 @@
+let firstName = document.getElementById('firstname');
+let lastName = document.getElementById('lastname');
+let phoneNumber = document.getElementById('phonenumber');
+let email = document.getElementById('email');
+let password = document.getElementById('password');
+let confirmPassword = document.getElementById('cpassword');
+let signupbutton = document.getElementById('signupbutton');
+
+// to trigger the required error messages when the user clicks the signup button
+signupbutton.addEventListener('click', () => {
+    let emailValid = validateEmail(email);
+    let numberValid = validateNumber(phoneNumber);
+    let passwordValid =  validatePassword(password);
+    let confirmPasswordValid = validateConfirmPassword(confirmPassword, password);
+    let firstNameValid = validateFirstName(firstName);
+    let lastNameValid = validateLastName(lastName);
+    if (emailValid && numberValid && passwordValid && confirmPasswordValid && firstNameValid && lastNameValid) {
```

Comment:
This check should be for debugging purpose. There is no need for this to be in your code